packages/data-stores/src/site/actions.ts

```diff
@@ -119,7 +125,21 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		return success;
 	}
 
+	function* setSiteTitle( title: string, siteId: number = window._currentSiteId ) {
```

Comment:
`window._currentSiteId` can be used indeed to get `siteId` inside the wpcom block editor but it's `undefined` in Calypso, Gutenboarding, etc. So I'd say it's not a good idea to have it as a default value for a required argument.